src/spaceone/monitoring/service/alert_service.py

```diff
@@ -65,7 +65,8 @@ def create(self, params):
 
         alert_vo = self.alert_mgr.create_alert(params)
 
-        self._create_notification(alert_vo, 'create_alert_notification')
+        if self._check_notification_urgency(alert_vo.urgency, project_alert_config_vo):
```

Comment:
An alert notification based on the alert's urgency is already checked in job progress.
So please remove this code.